Release v24.3.0-cycle3_27f712a (#463)
seqeralabs/docs: changelog/seqera-cloud/v24.3.0-cycle3_27f712a.mdx

---
title: Seqera Cloud v24.3.0-cycle3_27f712a
date: 2025-02-10
tags: [seqera cloud]
---

### Feature updates and improvements

- Studios:

>     - Implement a UI element that displays the currently applied date filter on the Dashboard page.

Reviewer: ```suggestion
    - Display the applied date filter on the Dashboard page.
```

    - Add support for custom templates for input control options such as workspace visibility and session lifespan.
- Fusion:
    - Default to Amazon Linux 2023 AMI when Fusion v2 is enabled.

>     - Add support for Fusion licensing mechanism.

Reviewer: ```suggestion
    - Add support for Fusion licensing.
```

- (Enterprise only) Add a job to save usage metrics to files so usage data can be accessed and shared by users even when they’re not connected to License Manager.
- Implement `FaviconService` that changes the favicon based on pipeline run or Studio status.
- Increase `accessTokenExpirationTime` by 10% to always trigger a token refresh when Platform is open in a browser tab that isn't hibernated.
- Allow Nextflow configuration parameters with embedded references to other parameters to be shown verbatim.
- Extend zombie jobs detection on all workflows reaching one of the terminal states (`SUCCEEDED`, `FAILED`, `CANCELLED`).

### Bug fixes 

> - Fixed an additional reference to `mainScript` parameter in a `pipeline-info` call.

Reviewer: ```suggestion
- Fixed an additional reference to the `mainScript` parameter in a `pipeline-info` call.
```
